scripts/test_tensor_api.py

```diff
@@ -54,188 +45,282 @@ class MooncakeStoreConfig:
     master_metrics_port: int
     check_server: bool
 
-    @staticmethod
-    def from_file() -> "MooncakeStoreConfig":
-        """Load the config from a JSON file."""
-        file_path = os.getenv(DEFAULT_MOONCAKE_CONFIG_PATH_ENV)
-        try:
-            with open(file_path) as fin:
-                config = json.load(fin)
-        except Exception as e:
-            raise RuntimeError(f"Failed to load config from {file_path}: {str(e)}")
-
-        return MooncakeStoreConfig(
-            local_hostname=config.get("local_hostname"),
-            metadata_server=config.get("metadata_server"),
-            global_segment_size=_parse_global_segment_size(
-                config.get("global_segment_size", DEFAULT_GLOBAL_SEGMENT_SIZE)
-            ),
-            # Zero copy interface does not need local buffer
-            local_buffer_size=DEFAULT_LOCAL_BUFFER_SIZE,
-            protocol=config.get("protocol", "tcp"),
-            device_name=config.get("device_name", ""),
-            master_server_address=config.get("master_server_address"),
-            master_metrics_port=config.get(
-                "master_metrics_port", DEFAULT_MASTER_METRICS_PORT
-            ),
-            check_server=config.get("check_server", DEFAULT_CHECK_SERVER),
-        )
-
     @staticmethod
     def load_from_env() -> "MooncakeStoreConfig":
-        """Load config from a file specified in the environment variable.
-        export MOONCAKE_MASTER=10.13.3.232:50051
-        export MOONCAKE_PROTOCOL="rdma"
-        export MOONCAKE_DEVICE=""
-        export MOONCAKE_TE_META_DATA_SERVER="P2PHANDSHAKE"
-        """
-        # other required environment variables...
+        """Load configuration from environment variables."""
         if not os.getenv("MOONCAKE_MASTER"):
-            raise ValueError("The environment variable 'MOONCAKE_MASTER' is not set.")
+            raise ValueError("Environment variable 'MOONCAKE_MASTER' is not set.")
         return MooncakeStoreConfig(
             local_hostname=os.getenv("LOCAL_HOSTNAME", "localhost"),
             metadata_server=os.getenv("MOONCAKE_TE_META_DATA_SERVER", "P2PHANDSHAKE"),
-            global_segment_size=_parse_global_segment_size(
+            global_segment_size=parse_global_segment_size(
                 os.getenv("MOONCAKE_GLOBAL_SEGMENT_SIZE", DEFAULT_GLOBAL_SEGMENT_SIZE)
             ),
-            # Zero copy interface does not need local buffer
             local_buffer_size=DEFAULT_LOCAL_BUFFER_SIZE,
             protocol=os.getenv("MOONCAKE_PROTOCOL", "tcp"),
             device_name=os.getenv("MOONCAKE_DEVICE", ""),
             master_server_address=os.getenv("MOONCAKE_MASTER"),
-            master_metrics_port=int(
-                os.getenv("MOONCAKE_MASTER_METRICS_PORT", DEFAULT_MASTER_METRICS_PORT)
-            ),
+            master_metrics_port=int(os.getenv("MOONCAKE_MASTER_METRICS_PORT", DEFAULT_MASTER_METRICS_PORT)),
```

Comment:
Don't need to rewrite it, import MooncakeStoreConfig from mooncake-wheel/mooncake